.gitignore

```diff
@@ -25,6 +25,8 @@ test-reports/
 package-lock.json
 .eslintcache
 /types
+types/
```

Comment:
Different than `/types` on line above? I think `types/` causes a problem if we create a dir called `types/` to hold our manual type definitions.

Reply:
I wasn't sure, probably is duplicate so I'll remove this one.